test/services/EmailServiceSpec.scala

```diff
@@ -73,6 +84,11 @@ class EmailServiceSpec extends BaseSpec with MockFactory {
       val result: Either[ErrorModel, EmailRendererResponseModel] = await(service.sendEmailRequest(messageModel))
       result shouldBe Right(EmailRendererResponseModel(ACCEPTED))
     }
+    "return a Left when an incorrect template id has been passed" in {
```

Comment:
This test is confusing - the template ID being used for the test - "VRT12A_SM9A" - is a valid template ID. The only reason this test fails is because it's a template ID that needs to be linked to an `EmailAddressChangeModel`, but you've used a generic `MessageModel`. I think it would make more sense to update the template ID in your model to be something unrecognised. Then the scenario is the same, but it should actually throw an exception, not return a Left. (this is the scenario talked about above and why the try catch block is needed in `EmailMessageService`)